### PR TITLE
Docs versioning

### PR DESCRIPTION
**Pull request by zachberry**
Adds plugins and updates Rakefile to manage versioning. Also adds release/latest URLs which redirect to the most recent version

### CODE REVIEW

**Comment by zachberry** [comment]
Note here - This https://github.com/ucfopen/Obojobo/pull/598 points to a specific release version. Since this PR introduces the /latest release redirect we could use that in the referenced issue

File: assets/css/doc.css [hunk 1]
```diff
@@ -733,12 +757,16 @@ blockquote {
 	background: #145bad;
 	color: #fff;
 	min-height: 50px;
-	padding: 8px 0;
+	/* padding: 8px 0; */
```
Comment (gitnix) [inline assets/css/doc.css]: comment

File: docs/README.md [hunk 2]
```diff
@@ -1,34 +1,64 @@
...
+- Bundler
+
+# Developing the docs
 
-# Running the HTML Proofer
+- Run `bundle exec rake dev`
+- Go to http://127.0.0.1:4000/Obojobo-Docs/ in your browser
 
-- Run `bundle exec rake test`
+Updating the files in the `./releases` directory should auto-update in your browser.
```
Comment (gitnix) [inline docs/README.md]: The files in the ./releases directory will auto-update in your browser when they are modified.

File: _includes/release_selector.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div class="styled-select">
```
[the remark this answers is not in the document]
Comment (zachberry) [reply _includes/release_selector.html]: Good point, I like reversing the order more

File: _layouts/default.html [hunk 4]
```diff
@@ -1,159 +1,193 @@
...
+							alt="Obojobo"
+						/>
+						<h2 class="headerTitleWithLogo">Obojobo</h2>
+					</a>
+					{% include release_selector.html %}
+					<div class="navigationWrapper navigationSlider">
+						<nav class="slidingNav">
+							<ul class="nav-site nav-site-internal">
+								<!--
+									<li class="{% if current_menu == 'teachers' %}siteNavGroupActive{% endif %}"><a href="{{ site.baseurl }}/releases/v{{ page.docs_version }}/teachers/" target="_self">Teachers</a></li>
```
[the remark this answers is not in the document]
Comment (zachberry) [reply _layouts/default.html]: Was a note to ourselves that we want to add a teachers section. Removing, created https://github.com/ucfopen/Obojobo-Docs/issues/28

**Comment by zachberry** [comment]
@gitnix Could you go over the last commit, it should resolve your comments

**Comment by gitnix** [comment]
Also would be nice for 404s to suggest that a file you are looking for may not exist for blah version.

**Comment by zachberry** [comment]
@gitnix I like that idea. Want to defer to later though - made #29

**Pull request merged**